content/guides/developer-guides/delfin/_index.md

```diff
@@ -31,33 +31,183 @@ Delfin is an Infrastructure Management framework developed in Python programming
 SODA Delfin project already contains some [drivers](https://github.com/sodafoundation/delfin/tree/master/delfin/drivers), which can be used as reference by the new third party driver developers.
 
 Existing Delfin Drivers for reference:
-  - fake_driver - This is a dummy/sample driver used for testing purpose
-  - Huawei - This driver implements Huawei's OeanStor backend
-  - VMAX - This driver implements Dell EMC's VMAX storage backend
+- fake_driver - This is a dummy/sample driver used for testing purpose
+- Huawei - This driver implements Huawei's OeanStor backend
+- VMAX - This driver implements Dell EMC's VMAX storage backend
 
 ### Code changes needed
 
 * Add driver plugin 'entry points' to the file 'setup.py'.
+  ```python
+
+    'delfin.storage.drivers': [
+        ...
+        'sample_vendor sample_model = delfin.drivers.sample_vendor:SampleStorageDriver',
+        ...
+    ]
+
+  ```
 
 * Create driver source code folder under <delfin path>/delfin/drivers/
+  ```bash
+  mkdir -p /path-to-delfin/delfin/drivers/sample_vendor
+  touch /path-to-delfin/delfin/drivers/sample_vendor/__init__.py
+  ```
 
 * Extend base class StorageDriver defined in <delfin path>/delfin/drivers/driver.py, to implement a new driver.
 
+  ```python
+  # Copyright 2020 The SODA Authors.
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+
+
+  from delfin.drivers import driver
+
+
+  class SampleStorageDriver(driver.StorageDriver):
+    """SampleStorageDriver shows how to implement the StorageDriver,
+    it also plays a role as sample to fake data for being tested by clients.
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
For the new driver just extend the class or any modification can be done(in any case?), please mention if needed

Reply:
'Extend base class StorageDriver' is already mentioned in the steps